### PR TITLE
docs: remove duplicated container install instructions

### PR DESCRIPTION
**Pull request by ollytom**
This is maintained in the README (for now). Conflicting info can be
confusing; see https://github.com/meerkat-dashboard/meerkat/issues/105

### CODE REVIEW

**Comment by davekempe** [comment]
if I'm reading this right, there are no install instructions at all now? Methinks you have gone too far.

**Comment by ollytom** [comment]
Install instructions are in the README. Where the instructions live can be discussed separately (see https://github.com/meerkat-dashboard/meerkat/issues/82). Just removing the duplicate conflicting info here.

**Comment by ollytom** [comment]
FWIW I think general user install instructions should actually be on the website!

File: docs/index.md [hunk 1]
```diff
@@ -16,68 +14,10 @@ Or a video workflow from left to right:
...
-or
-```
-docker stack deploy -c docker-compose.yml meerkat
-```
-
-The default `docker-compose.yml` uses the directories `config/` `dashboards/` and `dashboards/` for data persistance.  Feel free to edit docker-compose.yml as you see fit though, of course.
-
-We have also included a Dockerfile in case you want to build your own image.
-
-### Build without Docker
```
Comment (davekempe) [inline docs/index.md:41]: @ollytom Did you mean to remove this bit?

**Review by davekempe — COMMENTED** [review]
see comment on line 41

File: docs/index.md (hunk 1 above)
Comment (ollytom) [reply docs/index.md:41]: Yes - this is already in the README (for now!)

**Comment by davekempe** [comment]
need to start again on the docs - cos we have a tarball method now and want to do a big refactor.

**Pull request closed without merging**